Source/Orts.Simulation/Simulation/Timetables/ProcessTimetable.cs

```diff
@@ -1760,6 +1761,10 @@ public void ExtractStartTime(string startString, string consistInfo, Simulator s
                                                     setConsistName = true;
                                                     break;
 
+                                                case "direction":
+                                                    createPoolDirection = String.Copy(thisQualifier.QualifierValues[0]);
```

Comment:
Tip for future: `String.Copy` is unnecessary, see the remarks on https://docs.microsoft.com/en-us/dotnet/api/system.string.copy. `createPoolDirection = thisQualifier.QualifierValues[0];` is the same logically.